process/interface.go

```diff
@@ -1088,10 +1088,17 @@ type EpochNotifier interface {
 	IsInterfaceNil() bool
 }
 
+// RoundNotifier can notify upon an epoch change and provide the current epoch
+type RoundNotifier interface {
+	RegisterNotifyHandler(handler vmcommon.RoundSubscriberHandler)
```

Comment:
RoundSubscriberHandler should be move to elrond-core

Reply:
it can stay like this now - add a TODO

Reply:
added todo